app/renderer/js/pages/preference/preference.ts

```diff
@@ -77,32 +81,32 @@ class PreferenceView extends BaseComponent {
 	}
 
 	// Handle toggling and reflect changes in preference page
-	handleToggle(elementName, state) {
+	handleToggle(elementName: string, state: boolean): void {
 		const inputSelector = `#${elementName} .action .switch input`;
 		const input = document.querySelector(inputSelector);
 		if (input) {
-			input.checked = state;
+			(input as HTMLInputElement).checked = state;
 		}
 	}
 
-	registerIpcs() {
-		ipcRenderer.on('switch-settings-nav', (event, navItem) => {
+	registerIpcs(): void {
+		ipcRenderer.on('switch-settings-nav', (_event: Event, navItem: string) => {
```

Comment:
I have been meaning to turn off this rule that make use `_` before unused parameter. Can you look into this?

Reply:
We have done this while migrating other files as well. Will have to change it after everything is done later, if we want to do this.

Reply:
Actually, we can leave the already done as is with `_`, if we turn this lint rule off we just don't have to keep doing this.

Reply:
Ok, we can then turn this off in a later commit!